crates/api/src/lib.rs

```diff
@@ -59,7 +59,10 @@ pub use timestamp::*;
 
 pub mod fetch;
 
-pub mod op_store;
```

Comment:
Prefer not having both "pub mod" and "pub use". I only need to be able to import it one way. If we have conflicts between modules then we'd have to resolve that with "pub use" anyway